SolarParkService/src/main/java/org/energygrid/east/solarparkservice/rabbit/RabbitProducer.java

```diff
@@ -0,0 +1,71 @@
+package org.energygrid.east.solarparkservice.rabbit;
+
+import com.rabbitmq.client.*;
+
+import java.io.IOException;
+import java.util.UUID;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class RabbitProducer extends RabbitConnection {
+
+    private final static Logger logger = Logger.getLogger(RabbitProducer.class.getName());
```

Comment:
```suggestion
    private static final Logger logger = Logger.getLogger(RabbitProducer.class.getName());
```
Volgorde in Java best practices is eerst static dan final